### DISCUSSION TITLE
problem training custom model in gui 2.0

### DISCUSSION THREAD

**Discussion by WillFMitch** [birdnet-team/BirdNET-Analyzer#687] [General]
Hi there,

This is a copy paste of a question I asked on reddit about a week ago.

I just downloaded the new gui2.0. I trained a couple of custom models using both version 1.5 and version 2.0 and using identical training data and settings. These were for two separate species. Performance on the models trained using gui2.0 was very poor. Many hits with confidence score of 1.0 (!!!) from non target species. Similar examples from these non-targets have been included in the 'background' folder of training data.

Something seems to have gone awry with the custom model feature of the new version. 

Any suggestions?

**Comment by Josef-Haupt** [comment]
Hey, could you provide us with the settings used?
And, only if possible, with some of the training data, so that we an debug?

**Reply by WillFMitch** [reply]
Sure, I used entirely default settings for both gui1.5 and gui2.0.
autotone = no
epochs = 50
batch size = 32
learning rate = .001
upsampling = repeat
hidden units = 0
bandpass min/max 0, 15000
crop mode = centre
cache = none

I have attached a subset of the training data as the whole dataset is too large to be attached. The model trained on gui1.5 performed very well (1 false positive from a random sample of 100 detections with scores ranging from 0.5-0.99) As I said in my earlier post, the gui2.0 version had many false positives. 

I'm very appreciative of any help you can offer. The species is the Crested Bellbird Oreoica gutturalis
[training_sample.zip](https://github.com/user-attachments/files/20239332/training_sample.zip)

**Reply by max-mauermann** [reply]
You mentioned you use just the default parameters. But i noticed the default learning rate changed to 0.0001 in v2.0.
Could you try if you get similar results with the 0.001 learning rate in v2.0?

I also did some testing with the data you provided and also could see that the model trained with v2.0 had some higher confidence scores for the background samples then the one trained with v1.5. Although this was not as extreme as you described, the scores were still mostly <0.5.
